### PR TITLE
add group table functionality

### PR DESCRIPTION
**Pull request by 1Lukukas**

### CODE REVIEW

File: frontend/src/components/Navbar.jsx [hunk 1]
```diff
@@ -10,6 +9,9 @@ export default function Navbar() {
       <AppBar position="static">
         <Toolbar>
           <Button color="inherit">Login</Button>
+          <Button color="inherit">Login</Button>
```
Comment (edvyno1) [inline frontend/src/components/Navbar.jsx]: maybe make the buttons say different things, plan out what buttons we will need perhaps

File: frontend/src/index.js [hunk 2]
```diff
@@ -5,9 +5,9 @@ import Layout from "./components/Layout";
 import "./index.css";
 
 ReactDOM.createRoot(document.getElementById("root")).render(
-  <React.StrictMode>
-    <Layout>
-      <App />
-    </Layout>
-  </React.StrictMode>
+  // <React.StrictMode>
```
Comment (edvyno1) [inline frontend/src/index.js]: either uncomment or delete unused lines

**Comment by edvyno1** [comment]
looks good if you resolve the comments :)

File: frontend/src/index.js (hunk 2 above)
Comment (1Lukukas) [reply frontend/src/index.js]: resolved.

**Pull request merged**